WhisperAPI/WhisperAPI/Models/ContextItem.cs

```diff
@@ -0,0 +1,12 @@
+﻿using WhisperAPI.Models.NLPAPI;
+using WhisperAPI.Models.Queries;
+
+namespace WhisperAPI.Models
+{
+    public class ContextItem
```

Comment:
Nice, i like the name. Just add bool Relevant here like I said in an other comment